modular-docs-manual/content/topics/module_guidelines-procedure.adoc

```diff
@@ -32,12 +32,14 @@ The procedure consists of one or more steps required to complete the procedure.
 
 For single-step procedures, use an unnumbered bullet instead of a numbered list.
 
+NOTE: Not all numbered lists in documentation are procedures. You can also use numbered lists in any module type for non-procedural sequences, such as a process flow of system actions.
+
 [discrete]
-== Procedure verification steps
+== Procedure Verification
```

Comment:
I've made the title headline style strictly for consistency, even though it is against CCS convention of sentence style in headings. Issue #133 will eventually address the heading style issue throughout.

Reply:
Sentence case should be used. See https://github.com/redhat-documentation/modular-docs/issues/133
```suggestion
== Procedure verification
```